src/main/webapp/app/pages/curation/GeneListPage.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useEffect, useMemo, useState } from 'react';
```

Comment:
This is new and needs review.

Also note that we will moving the pagination/searching functionality of the table to the table component in a future PR (track by ticket: https://github.com/oncokb/oncokb/issues/3564)

Reply:
`OncokbTable` from the public repo does have sorting/pagination, but the table component in this repo does not. We are using react-table-v7 and public is using react-table-v6, so we can't completely copy over the table.

@zhx828 Do you think we should downgrade to v6 and just reuse the table from public repo?

I agree that we should have two table components, one for server-side and one for client side pagination/search. I will update the ticket

Reply:
Not familiar with v7. Any reason we upgraded?